### DISCUSSION TITLE
Any success with the SB100-100?

### DISCUSSION THREAD

**Discussion by VirtualKev** [GerryDazoo/Slinger#508] [Q&A]
Folks, I know my SB100-100 is ancient, but I held out hope... we have the right box password, but get the following when attempting to connect from the outside world (I portmapped 5365 to 5001... am I missing some ports or something dumb like that?):

Slinginfo  "Solo/Pro/ProHD" 12 30 24.xxx.xxx.244 5365 3000 10
8080 Starting Stream for  108.yyy.yyy.209:31438
Connecting... ('24.xxx.xxx.244', 5365) Control
cmd: 0x67 err: 0x2b 8
8080 Error Starting Session. Check your admin password in config.ini file!
8080 ERROR: Slingbox session startup failed.
8080 Logging Out

**Comment by RotatedBits** [comment]
I had two SB100-100 units working with Slinger prior to the Sling Server Shutdown and within a few days after, Slinger started reporting the same errors.  It appears my two SB100-100 units (and possibly yours) have been "Bricked" by Sling.  When looking at the boxes' Information tab in Slingplayer Desktop, it shows "Streaming Disabled".  To date, I have not been able to recover either SB100-100 by reset, factory reset, etc.  I am also hoping someone finds a way to unbrick this model slingbox.

For what its worth - I bought a ProHD on eBay and was easily able to get Slinger working with it on a Raspberry Pi 3B+.  

A million thanks to **Gerry** for all the great work on Slinger :-)

**Reply by sp222222** [reply]
you guys don't read much do you..... block OUTGOING PORT 5678 
if you did not then you now have a bricked SB100 like I do.

**Comment by Toei79** [comment]
i have one too but i dont connnected in years, doing safe steps maybe work?
